UDPClientGUI/ViewModels/StringToColorConverter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using System;
+using System.Globalization;
+using System.Windows.Data;
+using System.Windows.Media;
+
+namespace UDPCommGUI.ViewModels
+{
+    class StringToColorConverter : IValueConverter
+    {
+        public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            return value.ToString() == "Connected" ? new SolidColorBrush(Colors.Green) : new SolidColorBrush(Colors.Red);
+        }
+
+        public object ConvertBack(object value, Type targetType, object parameter, CultureInfo culture)
+        {
+            if(value is SolidColorBrush)
+            {
+                return ((SolidColorBrush)value).Color == Colors.Red ? "Disconnected" : "Connected";
+            }
+            return "Disconnected";
```

Comment:
Nothing wrong with writing code this way.

But try to simplify it into one statement using the potentially null operator `?.`

Reply:
Rewritten to make easier to follow